engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/NkGlobalParameters.scala

```diff
@@ -11,6 +11,8 @@ import pl.touk.nussknacker.engine.api.ProcessVersion
 //Also, those configuration properties will be exposed via Flienk REST API/webconsole
 case class NkGlobalParameters(buildInfo: String,
                               processVersion: ProcessVersion,
+                              processOriginalName: String,
+                              namespace: Option[String],
```

Comment:
I wonder if it would be better to keep here only core params and for extensions add something like `rawParameters` here and extract typed object in place of extension. WDYT? Please consider situation: someone want to have some weird naming scenario like: name + namespace + mainPart. To achive that it will add here one more parameter `name: Option[String]`? Or better one: he want to have namespace as a list.

Reply:
Yes, it should be clearer this way, I'll change it